examples/simple/variables.tf

```diff
@@ -5,3 +5,19 @@ variable "logs_bucket" {
 variable "region" {
   type = string
 }
+
+variable "s3_key_prefix" {
+  type = string
+}
+
+variable "trail_name" {
+  type = string
```

Comment:
Why not default this to `cloudtrail` so that its backwards compatible?

Reply:
I got caught by this too. This is in the *examples*, not the `variables.tf`. The default is still `cloudtrail`, in this example it's not changing that, so it stays the same.

Reply:
Doh. Well there we go. I'm not used to all these examples!